jsons/serializers/default_date.py

```diff
@@ -0,0 +1,13 @@
+from datetime import date
+from jsons._date_impl import to_str, RFC3339_DATETIME_PATTERN
+
+
+def default_date_serializer(obj: date, **kwargs) -> str:
+    """
+    Serialize the given datetime instance to a string. It uses the RFC3339
+    pattern.
+    :param obj: the datetime instance that is to be serialized.
+    :param kwargs: not used.
+    :return: ``date`` as an RFC3339 string.
```

Comment:
```suggestion
    :return: ``obj`` as an RFC3339 date string.
```
That line is also wrong in the docstring of ``datetime_serializer``.